projects/convai2/baselines/seq2seq/eval_ppl.py

```diff
@@ -43,23 +45,14 @@ def next_word_probability(self, observation, partial_out):
         e.g.
         {'text': 'Run test program.'}, ['hello'] => {'world': 1.0}
         """
-        if not hasattr(self, 'prev_enc'):
-            self.prev_enc = None
-            self.last_text = None
-        if observation['text'] != self.last_text:
```

Comment:
is it a big speed difference? you could definitely keep this by simply storing the last text as before, i dont see the difference?

Reply:
fixed!